crime-means-assessment/src/main/resources/schemas/apiCreateMeansAssessmentResponse.json

```diff
@@ -34,6 +46,10 @@
       "type": "number",
       "description": "Upper Threshold Value from Ref Data"
     },
+    "fullThreshold": {
+      "type": "number",
+      "description": "Full Threshold Value from Ref Data"
+    },
     "result": {
       "type": "string",
       "description": "Outcome of an Assessment"

```

Comment:
Please also move the fullAssessmentAvailable field to be inside of the properties so that the whole schema is correct

Reply:
done